src/perfume.ts

```diff
@@ -191,9 +203,11 @@ export default class Perfume {
    * https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming/navigationStart
    */
   public getFirstPaint() {
+    // this method is only used once
     if (performance) {
       const navTiming = performance.timing;
       if (navTiming && navTiming.navigationStart !== 0) {
+        // why not performance.now()?
         return Date.now() - navTiming.navigationStart;
```

Comment:
I used http://msdn.microsoft.com/ff974719 style, but maybe `performance.now` is better.